packages/story-editor/src/components/videoTrim/useVideoTrimMode.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useFeature } from 'flagged';
+import { useCallback, useMemo } from '@web-stories-wp/react';
+
+/**
+ * Internal dependencies
+ */
+import { useCanvas, useStory } from '../../app';
+
+function useVideoTrimMode() {
+  const isVideoTrimEnabled = useFeature('enableVideoTrim');
+  const { isEditing, isTrimMode, setEditingElementWithState, clearEditing } =
+    useCanvas(
+      ({
+        state: { isEditing, editingElementState: { isTrimMode } = {} },
+        actions: { setEditingElementWithState, clearEditing },
+      }) => ({
+        isEditing,
+        isTrimMode,
+        setEditingElementWithState,
+        clearEditing,
+      })
+    );
+  const { selectedElement } = useStory(({ state: { selectedElements } }) => ({
+    selectedElement: selectedElements.length === 1 ? selectedElements[0] : null,
+  }));
+  const { isTranscoding, isMuting, isTrimming, local } =
+    selectedElement.resource;
```

Comment:
```suggestion
    selectedElement?.resource;
```

Reply:
I am getting a fatal error.

![Uploading Screenshot 2021-09-08 at 18.17.32.png…]()

Reply:
Fixed in 8162d8f69a.